src/assets/styles/abstracts/_mixins.scss

```diff
@@ -28,3 +28,29 @@
         @content;
     }
 }
+
+// Brand component focus & hover style overrides
+@mixin brand-focus-hover() {
+    background: unset;
+    border-radius: unset;
+    margin-inline: unset;
+    outline: solid 0.3125rem $darkRed;
+}
+
+// Menu link focus & hover style overrides
+@mixin menu-link-focus-hover-narrow() {
+    background-color: var(--fl-linkColor, $menuExpandedSelectedBGColour) !important;
+    border-radius: unset;
+    color: var(--fl-bgColor, $menuExpandedSelectedColour) !important;
+    margin: unset;
+    outline-color: transparent;
+    padding-block: unset;
+}
+
+@mixin menu-link-focus-hover-wide() {
+    background-color: inherit !important;
+    border-bottom: 0.625rem solid $menuExpandedSelectedBGColour;
+    color: var(--fl-linkColor, currentColor) !important;
```

Comment:
I don't suppose it's possible to remove the use !important? Perhaps a comment to explain what they are for would help if they cannot be removed.